restonomer-docs/source/restonomer_transformations.md

```diff
@@ -334,3 +334,102 @@ The transformed response will have the replaced value or pattern in the desired
   "col_C": 3.4678
 }
 ```
+
+## CastNestedColumn
+
```

Comment:
I think you have repeated this transformation. Now we have same description for CastNestedColumn twice in the documentation. Kindly remove this.